L/libosmium/build_tarballs.jl

```diff
@@ -39,7 +39,7 @@ dependencies = [
     BuildDependency(PackageSpec(name="Zlib_jll", uuid="83775a58-1f1d-513f-b197-d71354ab007a")),
     Dependency("Expat_jll"; compat="2.2.10"),
     Dependency("Bzip2_jll"; compat="1.0.8"),
-    Dependency("boost_jll"; compat="=1.76.0"),
+    Dependency("boost_jll"; compat="=1.87.0"),
```

Comment:
Now that I'm looking at this more carefully, this is a header-only file, so I have the same comment as this one: https://github.com/JuliaPackaging/Yggdrasil/pull/5660#discussion_r993955189